api/include/mfxstructures.h

```diff
@@ -507,6 +507,22 @@ enum {
     MFX_PROFILE_AV1_MAIN                    = 1,
     MFX_PROFILE_AV1_HIGH                    = 2,
     MFX_PROFILE_AV1_PRO                     = 3,
+
+    MFX_LEVEL_AV1_2                         = 0,
```

Comment:
Value '0' is reserved for `MFX_LEVEL_UNKNOWN`. Please do mapping of public API's levels definition to specific `seq_level_idx` internally

Reply:
How about doing it similarly to HEVC in this enum, i.e.:
```
MFX_LEVEL_AV1_2                         = 20,
MFX_LEVEL_AV1_21                       = 21,
...
```

Reply:
Sure. I have changed it in the latest commit.